components/ui/scroll-arrow.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useCallback, useEffect, useState } from "react";
```

Comment:
```suggestion
import { useCallback, useEffect, useState } from "react";
```

Nitpick: We no longer need to import `React` if we already imported another function.